cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -5827,11 +5827,85 @@ search = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")
 | `cs_mssql_user_changes_filter`
 action.cyences_notable_event_action = 1
 action.cyences_notable_event_action.param.filter_macro_name = cs_mssql_user_changes_filter
-action.cyences_notable_event_action.contributing_events = `cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json") class_type_desc="SQL LOGIN" | rename properties.* as *
+action.cyences_notable_event_action.contributing_events = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SQL LOGIN","SQL USER") | rename properties.* as *
 action.cyences_notable_event_action.system_compromised_search = | stats count by host
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json") class_type_desc="SQL LOGIN" host=$row.host$ | rename properties.* as *
+action.cyences_notable_event_action.system_compromised_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SQL LOGIN","SQL USER") host=$row.host$ | rename properties.* as *
 action.cyences_notable_event_action.attacker_search = | stats count by user
-action.cyences_notable_event_action.attacker_drilldown = `cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json") class_type_desc="SQL LOGIN" user=$row.user$ | rename properties.* as *
+action.cyences_notable_event_action.attacker_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SQL LOGIN","SQL USER") user=$row.user$ | rename properties.* as *
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = MSSQL
+
+[MSSQL - Role Changes]
```

Comment:
Let's add all 3 alerts for Oracle database as well. (it is okay if it is not 100% accurate, we can fix it when we find issue in different environment)